api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -0,0 +1,145 @@
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import AnalyzerRunException, FileAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class CAPEsandbox(FileAnalyzer):
+    max_tries: int = 25
+    to_respond = {}
+
+    def set_params(self, params):
+        self.__token = self._secrets["token"]
+        self.__vm_name = params.get("VM_NAME", "win7x64_8")
+        self.BASE_URL = self._secrets.get("url_key_name", "https://www.capesandbox.com")
+
+    def run(self):
+        API_URL: str = self.BASE_URL + "/apiv2/tasks/create/file/"
+        self.headers = {
+            "Authorization": f"Token {self.__token}",
+        }
+
+        files = {
+            "file": (self.filename, self.read_file_bytes()),
+        }
+
+        data = {"machine": self.__vm_name}
```

Comment:
since these are used only once, can just pass them directly to the `requests.post` function instead of defining explicitly before.

Reply:
Same applies for all the different `_url` variables before making request. You can pass them directly to the function. It reduces the number of code lines and makes the code more self-explanatory.